src/pages/gateway/work-with-mesh.md

```diff
@@ -159,3 +159,43 @@ For example:
 
 -  `aio config:delete console.project` Removes the current project from the cache.
 -  `aio config:delete console.workspace` Removes the current workspace from the cache.
+
+## Aliasing
+
+In GraphQL you can use aliasing to rename fields and avoid conflicts. This is particularly useful for API Mesh, because along with the [`rename`](../reference/transforms/rename.md) and [`prefix`](../reference/transforms/prefix.md) transforms it allows you to avoid naming conflicts.
```

Comment:
```suggestion
In GraphQL, you can use aliasing to rename fields and avoid conflicts. This is particularly useful for API Mesh, because along with the [`rename`](../reference/transforms/rename.md) and [`prefix`](../reference/transforms/prefix.md) transforms, it allows you to avoid naming conflicts.
```

I think you need to state how to define an alias. Based on this one example, it looks like the syntax is &lt;alias&gt;:&lt;default-name&gt;.

I would also like to see an explanation of when you'd use rename/prefix/alias. Since you don't mention an alias transform, it seems that you can use them without altering your mesh?

Reply:
aliasing is just standard GraphQL. i can add a link to their site. e.g. https://graphql.org/learn/queries/#aliases